Manual/manJHUGenerator.tex

```diff
@@ -24,14 +24,17 @@
 \large
 Manual for the JHU generator and MELA package
 }
+\author{I. Anderson, S. Bolognesi, F. Caola, J. Davis, Y. Gao, A. V. Gritsan, L. S. Mandacaru Guerra, Z. Guo, C. B. Martin, T. Martini, K. Melnikov, 
+R. Pan, M. Panagiotou, R. Rontsch, J. Roskes, U. Sarica, M. Schulze, M. V. Srivastav, N. V. Tran, A. Whitbeck, M. Xiao, Y. Zhou}
 \maketitle
 \begin{center}
 \small
 For simulation, reweighting, and analysis of single-produced resonances at hadron colliders \\
-(version v7.5.2, release date October 1, 2022) \\
+(version v7.5.3, release date August 25, 2023) \\
```

Comment:
This line and L1483 need to be updated once the library issue is resolved.

Reply:
It is unfortunate that there was a problem with the script: I did my part of uploading libraries correctly, but the libraries passed to me all appeared to correspond to the same architecture (I did not check, but identical file size 
should have been a red flag). My suggestion on Friday was to upload at least one library which is needed for the build check, and this way we could have the loop closed, even if other libraries come later (something I call logistics). Unfortunately that one was missing. In any case, this is beyond my control, we try to act quickly, but there is no promise things would be done over the weekend. About the other two comments to update the date and the release note: This is optional, depending if Jeff wants to bother doing that. Everything was ready on Friday, so if libraries get updated on Monday, I would not bother with the date change. The same about the comment in release notes: we never go to this level of detail in any of the release notes, so I would even suggest not to do that, but I would not object if Jeff wants to change.

Reply:
> It is unfortunate that there was a problem with the script: I did my part of uploading libraries correctly, but the libraries passed to me all appeared to correspond to the same architecture (I did not check, but identical file size should have been a red flag). My suggestion on Friday was to upload at least one library which is needed for the build check, and this way we could have the loop closed, even if other libraries come later (something I call logistics). Unfortunately that one was missing. In any case, this is beyond my control, we try to act quickly, but there is no promise things would be done over the weekend. About the other two comments to update the date and the release note: This is optional, depending if Jeff wants to bother doing that. Everything was ready on Friday, so if libraries get updated on Monday, I would not bother with the date change. The same about the comment in release notes: we never go to this level of detail in any of the release notes, so I would even suggest not to do that, but I would not object if Jeff wants to change.

I have to say I have a different opinion on the last comment. In the way it is written, nobody knows what is fixed, how it is fixed, and what is affected. If something is fixed, it is better to not leave the users guessing.

On uploading libraries, this should have been done at the same time MELA PR is closed. In the future, I will be explicitly requiring this step to be complete, and completely tested, before a MELA PR is accepted. Moreover, we cannot tell people to use a particular version of slc/gcc combination - these use cases are determined by analysis framework needs. If we start making requirements on compilers, even if temporary, we render our package useless, and would eventually lose our users.

Reply:
Your opinion is understood, but it is opposite to the long history of releases. You also misunderstood my comment: all libraries must be ready before we release the next version officially. However, PR may be accepted as long as CircleCI checks are successful. These are two separate things. Not each PR ends up with a public release. In any case, most of discussion today should have been started outside of git, we all know what needs to be done.